Doc/library/idle.rst

```diff
@@ -557,7 +557,7 @@ IDLE-console differences
 
 As much as possible, the result of executing Python code with IDLE is the
 same as executing the same code in a console window.  However, the different
-interface and operation occasionally affects visible results.  For instance,
+interface and operation occasionally affect visible results.  For instance,
```

Comment:
I wondered about that and could read it both ways, but I changed it because the first time I read it, it didn't feel right.  And, after thinking about it (before the commit), my reasoning was that 'interface and operation' doesn't seem like an 'it', but more like a 'they'.  But, if you meant it as one thing, then I can change it back.